### PR TITLE
12.0 beesdoo easymycoop fix

### PR DESCRIPTION
**Pull request by houssine78**
this PR contains fixes for this [task](https://gestion.coopiteasy.be/web#id=4581&view_type=form&model=project.task&action=479)

### CODE REVIEW

File: beesdoo_easy_my_coop/__manifest__.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
     'data': [
         'views/res_company.xml',
+        'views/res_partner.xml',
```
Comment (vvrossem) [inline beesdoo_easy_my_coop/__manifest__.py]: remove since the file doesn't exist anymore

**Review by vvrossem — CHANGES_REQUESTED** [review]
Also bump version number :p

File: beesdoo_easy_my_coop/__manifest__.py (hunk 1 above)
Comment (houssine78) [reply beesdoo_easy_my_coop/__manifest__.py]: i put the xml file

**Review by remytms — APPROVED** [review]
C'est bon pour moi. @houssine Tu fusionnes ? :)

**Pull request merged**